static/app/components/modals/inviteMembersModal/useInviteModal.tsx

```diff
@@ -194,6 +194,7 @@ export default function useInviteModal({organization, initialData, source}: Prop
       {
         organization,
         modal_session: sessionId.current,
+        is_new_modal: organization.features.includes('invite-members-new-modal'),
```

Comment:
Can we add the total count here? I understand that it would be the count that's attempted not necessarily the successful count.

And then where we display the error also add the log, because the error says how many failed already.